tests/cli/partition_test/multiple_partitions.bats

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bats
+
+TEST_HELPER_DIR="$BATS_TEST_DIRNAME/../test_helper"
+export TEST_HELPER_DIR
+
+load "$TEST_HELPER_DIR"/tools_check.bash
+load "$TEST_HELPER_DIR"/fluvio_dev.bash
+load "$TEST_HELPER_DIR"/bats-support/load.bash
+load "$TEST_HELPER_DIR"/bats-assert/load.bash
+
+setup_file() {
+    PRODUCE_CONSUME_MULTIPLE_PARTITIONS_TOPIC_NAME=$(random_string)
+    export PRODUCE_CONSUME_MULTIPLE_PARTITIONS_TOPIC_NAME
+    debug_msg "Topic name: $PRODUCE_CONSUME_MULTIPLE_PARTITIONS_TOPIC_NAME"
+
+    MULTI_LINE_FILE_NAME=$(random_string)
+    export MULTI_LINE_FILE_NAME
+
+    for i in {1..4}
+    do
+        echo $i >> "$MULTI_LINE_FILE_NAME"
+    done
+}
+
+teardown_file() {
+    echo "Tearing down, shutting down cluster components"
+    "$FLUVIO_BIN" topic delete "$PRODUCE_CONSUME_MULTIPLE_PARTITIONS_TOPIC_NAME"
+}
+
+@test "Create a topic for P/C Multiple Partitions" {
+    echo "Creates Topic: $PRODUCE_CONSUME_MULTIPLE_PARTITIONS_TOPIC_NAME for P/C Multiple Partitions"
+    run timeout 15s "$FLUVIO_BIN" topic create "$PRODUCE_CONSUME_MULTIPLE_PARTITIONS_TOPIC_NAME" --partitions 2 --replication 2
```

Comment:
make partitions and replication configuration. this should be passed from env variable so we can configure this on CI.
for this test, we can set replication as 1

Reply:
Applied: 1a47a91